stix_shifter/stix_translation/src/unmapped_attribute_stripper.py

```diff
@@ -0,0 +1,76 @@
+from .patterns.pattern_objects import StartStopQualifier, ObservationExpression, \
+    ComparisonExpression, Pattern, CombinedComparisonExpression, CombinedObservationExpression
+from stix_shifter.stix_translation.src.exceptions import DataMappingException
+
+
+class UnmappedAttributeStripper:
```

Comment:
@delliott90 would that be possible to put this `unmapped_attribute_stripper.py` in the /utils folder? or we can create another utils inside the '/stix_translation/src/' ? just for cleanliness cause we have so many files in stix shifter.

Reply:
Yeah, I thought about putting it there but wasn't sure since it's translation-specific and thought it should go under the STIX_translation folder. Though STIX_translation could have its own utils folder and that could contain exceptions, STIX_pattern_parser, transformers, and the unmapped_attribute_stripper.